src/coreclr/vm/appdomain.hpp

```diff
@@ -41,22 +41,15 @@ class BaseDomain;
 class SystemDomain;
 class AppDomain;
 class CompilationDomain;
-class AppDomainEnum;
-class AssemblySink;
-class EEMarshalingData;
 class GlobalStringLiteralMap;
 class StringLiteralMap;
 class MngStdInterfacesInfo;
-class DomainModule;
 class DomainAssembly;
-struct InteropMethodTableData;
 class LoadLevelLimiter;
 class TypeEquivalenceHashTable;
-class StringArrayList;
 
 #ifdef FEATURE_COMINTEROP
-class ComCallWrapperCache;
-struct SimpleComCallWrapper;
+class RCWCache;
```

Comment:
Do we need this in the header or can it be moved to the compilation unit?

Reply:
We need it here for `m_pRCWCache` and its associated methods (L1960).